package.json

```diff
@@ -14,9 +14,9 @@
     "prettier:assemblyscript": "prettier --write ./assembly/**/*.ts",
     "prettier:ts": "prettier --write ./src.ts/**/*.ts ./tests/*.ts",
     "prettier": "npm-run-all prettier:assemblyscript",
-    "test": "ts-mocha --timeout 0 ./tests/**.spec.ts",
-    "test:view": "ts-mocha --timeout 0 ./tests/view.spec.ts",
-    "test:release": "ts-mocha --timeout 0 ./tests/release.spec.ts",
+    "test": "ts-mocha --timeout 0 --reporter spec ./tests/**.spec.ts",
```

Comment:
what does reporter do?

Reply:
needed for it to give a summary at the end to tell you what tests passed and failed